docs/concepts.rst

```diff
@@ -372,6 +372,63 @@ cases, we can disable in-memory caching:
         return f'Hello {subject}.'
 
 
+Non-Deterministic Entities
+............................
+
+.. versionadded:: 0.6.4
```

Comment:
I'm not sure how to bump this version. I changed the file but I suspect that's not the only thing I need to do here. Can we pair on this?

Reply:
Yeah, there's a separate process for doing releases. We do it separately from feature PRs. I'll walk you through it when we do the next release.

It's a little awkward to refer to a not-yet-existing release in the docs, especially since we don't know for certain if the next release will be 0.6.5 or 0.7.0. However, this text won't be visible in the stable docs until we ship our next release, so it's not a huge deal.

Longer term, it would probably be a good idea to maintain a proper changelog and some kind of release plan.